vendor/github.com/coreos/go-oidc/v3/oidc/jose.go

```diff
@@ -17,4 +13,5 @@ const (
 	PS256 = "PS256" // RSASSA-PSS using SHA256 and MGF1-SHA256
 	PS384 = "PS384" // RSASSA-PSS using SHA384 and MGF1-SHA384
 	PS512 = "PS512" // RSASSA-PSS using SHA512 and MGF1-SHA512
+	EdDSA = "EdDSA" // Ed25519 using SHA-512
```

Comment:
hmm... not a great sign that the one addition here isn't defined in the linked RFC